src/components/citizen-list-item/index.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
```

Comment:
why do we call it "citizen", instead of "member" (which we use in other places)?

Reply:
The global app name (from the user perspective) is Citizen. We just haven't ever picked it up in our code. But, in this particular case, I used this to match the component name used in Figma.